django/contrib/admin/static/admin/css/base.css

```diff
@@ -424,7 +428,7 @@ table thead th.sorted .sortoptions a {
     width: 14px;
     height: 14px;
     display: inline-block;
-    background: url(../img/sorting-icons.svg) 0 0 no-repeat;
+    mask-image: url(../img/sorting-icons.svg) 0 0 no-repeat;
```

Comment:
This needs more work, as-is it’ll just make the icons stop work altogether.

Reply:
Would mask: url(../img/sorting-icons.svg) 0 0 no-repeat; work? Or do I need to break the properties up?

Reply:
Hmm! I’ve never used `mask` :) It’s looking great [based on the MDN docs for it](https://developer.mozilla.org/en-US/docs/Web/CSS/mask). Give it a go and if it works, great? (and add `-webkit-mask:` to be safe with browser support). Re pseudo-element: I would say if we don’t have to, might as well do without?